### DISCUSSION TITLE
post route + field with base64 content

### DISCUSSION THREAD

**Discussion by dielandre** [fastapi/fastapi#11548] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
APP

from fastapi import FastAPI, Request
from fastapi.responses import JSONResponse

import logging
from starlette.middleware.base import BaseHTTPMiddleware
import binascii
import json
import base64

from modulo1.routes import router as modulo1_router

app = FastAPI()

class FixJSONMiddleware(BaseHTTPMiddleware):
    async def dispatch(self, request: Request, call_next):
        # Ler o corpo da requisição original
        body = await request.body()
        print(body)
        body_str = body.decode('ascii')
        print(body_str)
        
        # Remover barras invertidas extras e corrigir barras duplas dentro de valores
        corrected_body_str = body_str.replace(r'\\', '\\').replace(r'\/', '/')

        # Substituir o corpo da requisição com a versão corrigida
        request._body = corrected_body_str.encode('utf-8')

        # Continue com a próxima etapa do processamento da requisição
        response = await call_next(request)
        return response
    
app.include_router(modulo1_router, prefix="/wisedoc", tags=["wisedoc"])
app.add_middleware(FixJSONMiddleware)


route

@router.post("/laudos/integracao")
def integracao(aLaudo: Tlaudo, dados_usuario: dict = Depends(del_bearer_authorization)):
   tipo_id = dados_usuario.get("identtipo")
   usuario_id = dados_usuario.get("identvalor")
   return atualizalaudo(aLaudo)

class

class Tlaudo(BaseModel):
    status: str
    accession_number: str
    paciente_id: str
    study_instance_uid: str
    laudo_conteudo: Optional[str]
    laudo_datahora: Optional[str]
    realizante_nome: Optional[str]
    realizante_crm: Optional[str]
    realizante_crm_uf: Optional[str]
    realizante_assinatura: Optional[str]
    revisor_nome: Optional[str]
    revisor_crm: Optional[str]
    revisor_crm_uf: Optional[str]
    revisor_assinatura: Optional[str]
```

### Description

I have a post route that receives a tlaudo class

This class has a field that receives base64 content

when the field comes with "//" fastapi truncates the body

json sent by client

{\"status\":\"L\" .20240426164328 \",\"report_content\":\"+Q6Wuqp8FCqwApdu3QBX3zmqafBtFfPuyyCu5MVCkXABCC//==\",\"report_datahora\":\"02/05/2024 08:56:12\",\"performer_name\":\"Radiologist Test\",\"realizador_crm\":\"40134\",\"realizador_crm_uf\":\"RS\",\"realizador_assinatura\":null,\"revisor_nome\":null,\"revisor_crm\": null,\"revisor_crm_uf\":null,\"revisor_assinatura\":null}

truncated json

INFO: Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)

report_conteudo\\":\\"+Q6Wuqp8FCqwApdu3QBX3zmqafBtFfPuyyCu5MVCkXABCC'
{\"status\":\"L\" .20240426164328 \",\"report_conteudo\":\"+Q6Wuqp8FCqwApdu3QBX3zmqafBtFfPuyyCu5MVCkXABCC

INFO: 127.0.0.1:51661 - "POST /wisedoc/laudos/integracao HTTP/1.1" 422 Unprocessable Entity

### Operating System

Windows

### Operating System Details

in Linux it also happens

### FastAPI Version

0.110.2

### Pydantic Version

2.7.0

### Python Version

Python 3.12.1

**Comment by achempak-polymer** [comment]
This is invalid json. Specifically this part:
```
"status":"L" .20240426164328 "
```
I'm not sure if the quote after the L is intentional or not, but if it is, then it needs to be escaped:
```json
{
   "status":"L\" .20240426164328 ",
   "report_content":"+Q6Wuqp8FCqwApdu3QBX3zmqafBtFfPuyyCu5MVCkXABCC//==",
   "report_datahora":"02/05/2024 08:56:12",
   "performer_name":"Radiologist Test",
   "realizador_crm":"40134",
   "realizador_crm_uf":"RS",
   "realizador_assinatura":null,
   "revisor_nome":null,
   "revisor_crm":null,
   "revisor_crm_uf":null,
   "revisor_assinatura":null
}
```

**Reply by dielandre** [reply]
I pasted the wrong json, sorry

json correct

{
   "status":"L\",
   "report_content":"+Q6Wuqp8FCqwApdu3QBX3zmqafBtFfPuyyCu5MVCkXABCC//==",
   "report_datahora":"02/05/2024 08:56:12",
   "performer_name":"Radiologist Test",
   "realizador_crm":"40134",
   "realizador_crm_uf":"RS",
   "realizador_assinatura":null,
   "revisor_nome":null,
   "revisor_crm":null,
   "revisor_crm_uf":null,
   "revisor_assinatura":null
}

the problem is where it is //

fastapi truncates, and postman also truncates

![example](https://github.com/tiangolo/fastapi/assets/34354135/b1d047b2-70b4-474c-b33f-ce332b25f1f8)

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
